injected/src/features/broker-protection/captcha-services/captcha.service.js

```diff
@@ -43,7 +43,7 @@ export function getSupportingCodeToInject(action) {
         return createError(`could not find captchaProvider with type ${captchaType}`);
     }
 
-    return SuccessResponse.create({ actionID, actionType, response: { code: captchaProvider.getSupportingCodeToInject() } });
+    return SuccessResponse.create({ actionID, actionType, response: { injectedCode: captchaProvider.getSupportingCodeToInject() } });
```

Comment:
Not strictly necessary, but decided to call this injectedCode on the receiving side, seemed clearer.